frame/contracts/src/weights.rs

```diff
@@ -206,11 +206,11 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Contracts CodeStorage (r:1 w:1)
 	// Storage: Contracts PristineCode (r:0 w:1)
 	fn instantiate_with_code(c: u32, s: u32, ) -> Weight {
-		(479_578_000 as Weight)
+		(680_527_000 as Weight)
```

Comment:
This is some non trivial weight change here (it happens to all benchmarks). Any idea why this is?

Reply:
This sounds like the changes that @joao-paulo-parity made are not insignificant then.